src/naive/datetime/mod.rs

```diff
@@ -126,6 +126,27 @@ impl NaiveDateTime {
         let datetime = NaiveDateTime::from_timestamp_opt(secs, nsecs);
         datetime.expect("invalid or out-of-range datetime")
     }
+    /// Creates a new [NaiveDateTime] from milliseconds since the UNIX Epoch.
```

Comment:
potentially the capitalization should be aligned between this and the next line, I'd probably go for lowercase in both

Reply:
Done